docs/ReactiveSwift.md

```diff
@@ -1,22 +1,22 @@
 # ReactiveSwift
 
-Moya provides an optional `ReactiveSwiftMoyaProvider` subclass of
+Moya provides an optional `ReactiveSwift` implementation of
 `MoyaProvider` that does a few interesting things. Instead of
 calling the `request()` method and providing a callback closure
 to be executed when the request completes, we use `SignalProducer`s
 (`RACSignal`s are also available for those who need it).
 
-A `ReactiveSwiftMoyaProvider` can be created much like a
+A `ReactiveSwift`-based `MoyaProvider` can be created from a
```

Comment:
I'd change that a little bit. It feels a bit odd to me. Maybe something along the lines:
> To use reactive extensions you don't need any additional setup. Just use your `MoyaProvider` instance.